### PR TITLE
Slider small bugs #425. focus on mouse click, if keyboard navigation …

### LINKED ISSUE #425: Slider small bugs
- [ ] Thumb should be focused on keyboard navigation.
Here's some details:
When I click on thumb (round thing on slider), then I press left - I don't see focus.
When I click on thumb, then press tab (which moves me to another thumb) - I see it being focused.
Should be focused in both cases.
image

- [ ] When I use keyboard navigation to trigger thumb - transition shouldn't happen.
image

### PR DESCRIPTION
**Pull request by igorprodev1**
closes #425

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <div
```
Comment (asvae) [inline packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:53]: Not related to the issue, but doesn't work well with negative values.
image

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue [hunk 2]
```diff
@@ -64,8 +64,8 @@
           :style="dottedStyles[0]"
           @mousedown="(moveStart($event, 0), setMouseDown($event, 1))"
           @touchstart="moveStart($event, 0)"
-          @focus="KeyboardOnlyFocusMixin_onFocus($event, 1), currentSlider = 0"
-          @blur="isKeyboardFocused = false"
+          @focus="isFocused = true, currentSlider = 0"
```
Comment (asvae) [inline packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:67]: Pretty sure you want to keep mixin.

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue [hunk 3]
```diff
@@ -245,14 +244,14 @@ export default class VaSlider extends Mixins(
...
-        { 'va-slider__handler--inactive': !this.hasMouseDown },
+        { 'va-slider__handler--inactive': !this.isFocused },
+        { 'va-slider__handler--inactive': !this.isFocused },
       ]
     }
 
     return {
-      'va-slider__handler--on-focus': !this.range && (this.flag || this.isKeyboardFocused),
-      'va-slider__handler--inactive': !this.hasMouseDown,
+      'va-slider__handler--on-focus': !this.range && (this.flag || this.isFocused),
```
Comment (asvae) [inline packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:253]: Focus and keyboard focus are kinda different things.
Focus we don't really want to visualize in any way for slider. Keyboard focus - we do.

**Review by asvae — COMMENTED** [review]
Couple of things to work on.

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue (hunk 1 above)
Comment (igorprodev1) [reply packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:53]: will fix

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue [hunk 4]
```diff
@@ -53,7 +53,7 @@
         <div
           ref="process"
           class="va-slider__track va-slider__track--selected"
```
[the remark this answers is not in the document]
Comment (igorprodev1) [reply packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:55]: will fix

**Review by asvae — APPROVED** [review]

File: packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (asvae) [reply packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue:55]: @igorprodev1 you can resolve convo if the issue is fixed. No need to wait for me etc.

**Pull request merged**